asyncio/core.py

```diff
@@ -232,6 +257,11 @@ def run_until_complete(main_task=None):
 
 # Create a new task from a coroutine and run it until it finishes
 def run(coro):
+    """Create a new task from the given coroutine and run it until it completes.
+
+    Returns the value returned be *coro*.
```

Comment:
should this be `...returned by *coro*` ?

Reply:
Ah, yes, thank you@!